<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputDialog</class>
 <widget class="QDialog" name="InputDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>480</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1. Enter or choose the file path:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtInputFilePath">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2. Select eye position:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioLeft">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>LEFT</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">radioBtnGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioRight">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>140</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>RIGHT</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">radioBtnGroup</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>541</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3. Enter dataset code and load the file that contains the list of imagery</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Query</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Code</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>File Name</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtCode4Query">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtFileList4Query">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLoading4Query">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>260</y>
     <width>41</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtCode4Target">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtFileList4Target">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLoading4Target">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>300</y>
     <width>41</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Target</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>4. Enter the output file name:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtOutputFileName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>541</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLoadingFilePath">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>60</y>
     <width>41</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnLoading4Query</sender>
   <signal>clicked()</signal>
   <receiver>InputDialog</receiver>
   <slot>btnLoadingQueryClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>550</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnLoading4Target</sender>
   <signal>clicked()</signal>
   <receiver>InputDialog</receiver>
   <slot>btnLoadingTargetClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>550</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>InputDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>InputDialog</receiver>
   <slot>btnAcceptClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnLoadingFilePath</sender>
   <signal>clicked()</signal>
   <receiver>InputDialog</receiver>
   <slot>btnLoadingFilePathClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>550</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btnLoadingQueryClicked()</slot>
  <slot>btnLoadingTargetClicked()</slot>
  <slot>btnAcceptClicked()</slot>
  <slot>btnLoadingFilePathClicked()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="radioBtnGroup"/>
 </buttongroups>
</ui>
